server/routes/commentRoutes.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+import commentController from "../controllers/commentController";
+
+
+router.get('/allComments', commentController.allComments);
+// router.post('/addComment', commentController.saveComment);
```

Comment:
Could you please remove the comment codes

Reply:
> Could you please remove the comment codes

>Removed all commented lines

Reply:
that file was outdated and i resolved the conflicts remained